sauce/features/budget/display-goal-amount/index.js

```diff
@@ -25,18 +25,19 @@ export default class DisplayTargetGoalAmount extends Feature {
       const targetBalance = subCategory.get('targetBalance');
       const targetBalanceDate = monthlySubCategoryBudgetCalculation.get('goalTarget');
       if (goalType === 'MF') {
-        $('#' + emberId + '.budget-table-row.is-sub-category div.budget-table-cell-goal').text('$' + monthlyFunding / 1000);
+        $('#' + emberId + '.budget-table-row.is-sub-category div.budget-table-cell-goal').text('$' + ynab.YNABSharedLib.currencyFormatter.format(monthlyFunding));
```

Comment:
actually, it would be best to use the helper function here because it will add the proper currency sign for you as well

`toolkitHelper.formatCurrency(monthlyFunding)`